indexer/lib/vendored/multiaddr.js

```diff
@@ -0,0 +1,58 @@
+// MIRRORED FROM https://github.com/filecoin-station/spark/blob/main/lib/multiaddr.js
```

Comment:
Time to package as a node module, or contribute upstream?

Reply:
IDK 🤷🏻 

There is an official package to convert multiaddr to URI, but it assumes that people want HTTP URI even for multiaddresses that are not HTTP (e.g. libp2p peers): https://www.npmjs.com/package/@multiformats/multiaddr-to-uri

I tried to use it, but it cannot be configured to do what we need.

I suppose I could contribute a pull request to add another configuration option to give us the behaviour we need. 🤔

Reply:
OTOH, this is only the second copy of this file, so maybe it's too early?